app/models/world_location_news_translation.rb

```diff
@@ -0,0 +1,11 @@
+class WorldLocationNewsTranslation
+  attr_accessor :translated_locales
+
+  def initialize(translated_locales = [])
+    @translated_locales = [].push(translated_locales).flatten
```

Comment:
Is this because `translated_locales` isn't always an array? If so you could use `Array(translated_locales)`.